generate_zeebe_initial_contact.py

```diff
@@ -0,0 +1,28 @@
+""""This script generates the initial contact string for Zeebe clusters in a multi-region setup."""
```

Comment:
I suggest also adding how this string is used (i.e. the name of the ENV var that is used by all Zeebe brokers)